custom_components/solarfocus/strings.json

```diff
@@ -200,6 +265,13 @@
           "1": "On"
         }
       },
+      "bo_charge" : {
```

Comment:
can't find that in the spec? looks like circulation?

Reply:
yes, it the pump to charge the warm water boiler with the hot water from the buffer

Reply:
```suggestion
      "bo_single_charge" : {
```